src/app/student-login/student-login.component.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../variables";
+
+
+.page{
+    background-repeat: no-repeat;
+    background-image: url('/assets/images/large-display/hill-compressed.jpg');
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.login-form{
+  min-width: 40%;
+  max-width: 60%;
+  max-height: 50%;
```

Comment:
```suggestion
```
Consider possibly removing this rule—hard-capping the size of the form to 50% won't resize the card inside. (The card will just spill over outside of the form. :man_shrugging:)

Reply:
(And if that happens, the alignment gets a bit wonky:
![image](https://user-images.githubusercontent.com/56209343/86419670-8d02a380-bc99-11ea-984d-3524bb8955c6.png)
)